src/Sylius/Component/Locale/Context/LocaleContext.php

```diff
@@ -32,6 +33,59 @@ class LocaleContext implements LocaleContextInterface
      */
     protected $storage;
 
+    /**
+     * Meta data for special languages,
+     * by default we assume that all languages are LTR except mentioned otherwise.
+     *
+     * TODO This list should be completed
+     *
+     * @var array
+     */
+    protected static $localeMetaData = array(
```

Comment:
Your right, it should be either extracted to our own library or imported from someone's library.